/* EYandexSerpTld represents the top-level domain of Yandex search page
   For example:
   YST_RU is set for requests from yandex.ru It is the default value.
   YST_TR is set for requests from yandex.com.tr
*/

enum EYandexSerpTld {
    YST_RU = 0;           /* ru */
    YST_UA = 1;           /* ua */
    YST_BY = 2;           /* by */
    YST_KZ = 3;           /* kz */
    YST_TR = 4;           /* tr */
    YST_COM = 5;          /* com */
    YST_UZ = 6;           /* uz */
    YST_AZ = 7;           /* az */
    YST_AM = 8;           /* am */
    YST_GE = 9;           /* ge */
    YST_KG = 10;          /* kg */
    YST_LV = 11;          /* lv */
    YST_LT = 12;          /* lt */
    YST_MD = 13;          /* md */
    YST_TJ = 14;          /* tj */
    YST_TM = 15;          /* tm */
    YST_EE = 16;          /* ee */
    YST_IL = 17;          /* il */
    YST_FR = 18;          /* fr */
    YST_DE = 19;          /* de */
    YST_FI = 20;          /* fi */
    YST_PL = 21;          /* pl */
    YST_EU = 22;          /* eu */
    YST_UNKNOWN = 1024;   /* unknown */
};
